landing-pages/site/data/committers.json

```
@@ -161,6 +161,12 @@
     "image": "https://github.com/vincbeck.png",
     "nick": "vincbeck"
   },
+  {
+    "name": "Wei Lee",
+    "github": "https://github.com/Lee-W",
+    "image": "https://github.com/Lee-W.png",
+    "nick": "weilee"
```

```suggestion
    "nick": "Lee-W"
```

This should be your github username. Might be worth renaming this lol.

---

Thanks! I though it could be any nick name haha. Justu updated it